### ISSUE TITLE
Feature Request: Use same window for PR if one is already open

### ISSUE THREAD

**Issue by teejae** [zenclabs/prmonitor#126]
When opening a PR, see if a tab already exists w/ the same PR open (files/convo view/etc), and focus that instead of opening new windows.

**Comment by mdvorsky** [comment]
For me (Chrome 74.0.3729.108, Mac OS 10.14.4), this appears to open tab N in the _current_ window even if the PR is opened in tab N a different window.

Steps to reproduce: 
 1. open a PR using the extension.
 2. Create a new chrome window
 3. try to open the same PR using the extension in the new window.

Actual behavior: nothing happens
Expected behavior: the window/tab with the PR open is brought into focus

**Comment by fwouts** [comment]
Thanks for flagging this @mdvorsky. I had assumed that `chrome.tabs.query()` would query across all windows, but it appears that's not the case.

Here is the current code for context: https://github.com/zenclabs/prmonitor/blob/master/src/tabs/implementation.ts#L38

Possible solution: https://stackoverflow.com/a/16185419/911298

Would you be interested in contributing a PR to fix this? Otherwise I can get to it in the next week or so.

**Comment by mdvorsky** [comment]
It appears to me that `chrome.tabs.query()` does indeed query all windows, but the problem is in the [`highlight`](https://github.com/zenclabs/prmonitor/blob/master/src/tabs/implementation.ts#L43) call as that only specifies tab index, but not windowId.

I'll see if I can figure out how to test this possible fix:
```
        chromeApi.tabs.highlight({
          windowId: existingTab.windowId,
          tabs: existingTab.index
        });
```

**Comment by mdvorsky** [comment]
OK, my fix suggested above "works", but it's not a complete fix. `tabs.highlight` will now focus the right tab in the other window, however, it does not switch the focus to that other window.

**Comment by fwouts** [comment]
Fixed in #189 and live in v0.3.5 :)
